wizeline/wize-q-remix
# feat: add creation of embeddings with openai

#### What does this PR do?
- Adds controllers and schema updates to support embeddings.
- Adds OpenAI integration of node. Currently the API key is not set, pending from IT. But not necesarry for merge.

#### How should this be manually tested?
- Clone this branch and verify prisma migrations work on your local.
- Answer a question and verify there are no errors


#### What are the relevant tickets?
Closes #242
